src/components/affix/index.js

```diff
@@ -59,9 +54,7 @@ export default class Affix extends PureComponent {
     }
 
     return (
-      <Animated.View style={[styles.container, containerStyle]}>
-        <Animated.Text style={[style, textStyle]}>{children}</Animated.Text>
-      </Animated.View>
+      <Text style={[styles.container, containerStyle, style, textStyle]}>{children}</Text>
```

Comment:
This change also means that we have to have the `label` always positioned above the input field, even in cases where it's not focused? Correct?

Reply:
No, the affix animation is now done on a parent container which also animates the text input. It shouldn't cause issues with the label, only downside is that the textinput and the affix can't be animated separately (but we currently don't have that usecase).